### PR TITLE
Translate spec/cloudevents/languages/zh-CN/RELEASE_NOTES to zh-CN

### PR DESCRIPTION
**Pull request by VedaGao**
Translate spec/cloudevents/languages/zh-CN/RELEASE_NOTES to zh-CN

#1027

### CODE REVIEW

**Comment by VedaGao** [comment]
@JieDing Please review this translation, thank you.

**Comment by JieDing** [comment]
> @JieDing Please review this translation, thank you.

OK, I'll review it asap

File: cloudevents/languages/zh-CN/RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,7 +2,163 @@
...
+- 阐明 HTTP header 值编码和解码要求 (#816)
+- 处理错误的入门指南 (#763)
+- 信息分类扩展 (#785)
+- 阐明分区扩展在 Kafka 中的作用 (#727)
+
+## v1.0.1 - 2020/12/12
+
+- 添加 protobuf 格式作为子协议 (#721)
+- 允许 JSON 值为 null，即未设置 (#713)
+- [Primer] 添加 Non-Goal w.r.t Security (#712)
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/RELEASE_NOTES.md:31]:
```suggestion
- 在入门文档中将安全性相关内容定为规范的非目标内容 (#712)
```

**Review by JieDing — COMMENTED** [review]
Thx for your efforts. Here are some feedbacks. @VedaGao

**Comment by duglin** [comment]
Please sign your commits to make the DCO check happy

**Comment by duglin** [comment]
Any update on this one?

**Comment by duglin** [comment]
@JieDing how's this one looking?

**Comment by JieDing** [comment]
> @JieDing how's this one looking?

LGTM

**Comment by duglin** [comment]
thanks - on the agenda for today

**Comment by duglin** [comment]
Approved on the 8/11 call

**Pull request merged**